### PR TITLE
Translating new lines in dutch

### PR DESCRIPTION
**Pull request by Vyolex**
Translating new lines that got introduced in #450.

### CODE REVIEW

**Comment by Vyolex** [comment]
Line 123, 124 and 125 not being changed is intentional.

Temp. is a valid abbreviation for "Temperatuur"
Accel. is a valid abbreviation for "Acceleratie"

File: gui/public/i18n/nl/translation.ftl [hunk 1]
```diff
@@ -87,10 +87,26 @@ navbar-settings = Instellingen
 bvh-start_recording = BVH opnemen
 bvh-recording = Opname bezig...
 
-## Overlay settings
+## Widget: Overlay settings
+widget-overlay = Overlay
 overlay-is_visible_label = Overlay in SteamVR weergeven
 overlay-is_mirrored_label = Overlay weergeven als spiegel
```
Comment (ImUrX) [inline gui/public/i18n/nl/translation.ftl:93]:
```suggestion
widget-overlay-is_visible_label = Overlay in SteamVR weergeven
widget-overlay-is_mirrored_label = Overlay weergeven als spiegel
```

**Review by ImUrX — CHANGES_REQUESTED** [review]
``settings-general-tracker_mechanics-amount`` is now ``settings-general-tracker_mechanics-filtering-amount``

**Comment by Vyolex** [comment]
> `settings-general-tracker_mechanics-amount` is now `settings-general-tracker_mechanics-filtering-amount`

should be fixed now

**Review by Louka3000 — APPROVED** [review]

**Review by ImUrX — APPROVED** [review]

**Pull request merged**